cmd/zoekt-sourcegraph-indexserver/merge.go

```diff
@@ -185,26 +190,17 @@ func (c *compound) add(cand candidate) {
 	c.size += cand.sizeBytes
 }
 
-func generateCompounds(shards []candidate, targetSizeBytes int64) []compound {
+func generateCompounds(shards []candidate, targetSizeBytes int64) ([]compound, []candidate) {
```

Comment:
A doc string for this function would now be nice.

Reply:
done